Tests/XcodeGenKitTests/ProjectSpecTests.swift

```diff
@@ -101,6 +101,16 @@ func projectSpecTests() {
                 try expectValidationError(spec, .invalidSchemeConfig(scheme: "scheme1", config: "debugInvalid"))
                 try expectValidationError(spec, .invalidSchemeConfig(scheme: "scheme1", config: "releaseInvalid"))
             }
+
+            $0.it("allows missing optional file", closure: {
```

Comment:
nitpick: we use trailing closure for these 👍 
Thanks for the test! :)

Reply:
Uniform style is important. Changed it.